### ISSUE TITLE
Provide custom localization files for DateInRegionFormatter

### ISSUE THREAD

**Issue by kevinrenskers** [malcommac/SwiftDate#380]
The `colloquialSinceNow` function is almost perfect, however for my project I need some of the date formatting to be slightly different. For example instead of `%d minutes ago` I need it to be `%d mins ago`.

How can I override the formatting strings, as defined in https://github.com/malcommac/SwiftDate/blob/develop/Sources/SwiftDate/SwiftDate.bundle/en-GB.lproj/SwiftDate.strings?

**Comment by malcommac** [comment]
Actually it's not supported unless you modify manually one of the locales.
However due to the fact we want also support Linux as target (see #352) we should need to provide an extra layer upon the NSLocalizedString() functions.
I'm working on a LocalizationHelper class which is able to load a custom file at specified path as translation language; it should be pretty easy to develop and it can also be used to set a custom file as localized .strings file which suits your need.
I'll post some updates on it in the following days, so keep an eye on this topic.

**Comment by malcommac** [comment]
Added `Localization` class.
Replaced `locale` property of `DateInRegion` with `localization` which is of type `Localization`.
By default it's initialized using the same `DateInRegion`'s `locale`.
You can however set it with the absolute path to a valid `.strings` file:

```swift
let formatter = DateInRegionFormatter()
formatter.localization = Localization(path: [PATH_TO_YOUR_STRINGS_FILE])
// use the formatter
```

It will be available since 4.0.12.
That's the reference to the commit https://github.com/malcommac/SwiftDate/commit/f1f16721f34f36798279012213d25942e60593ee.
